regression/cbmc/CMakeLists.txt

```diff
@@ -1,3 +1,3 @@
 add_test_pl_tests(
-    "$<TARGET_FILE:cbmc>" -X smt-backend
+    "$<TARGET_FILE:cbmc> --validate-goto-model" -X smt-backend
```

Comment:
I'm just extending #3661 to fix the stuff that was failing due to df616e4 in the interest of getting PRs merged, I'd suggest it might be better to enable `--validate-ssa` in a separate PR.